packages/desktop-client/src/components/FinancesApp.js

```diff
@@ -57,118 +57,82 @@ import PostsOfflineNotification from './schedules/PostsOfflineNotification';
 import Settings from './settings';
 import Titlebar, { TitlebarProvider } from './Titlebar';
 
-function PageRoute({
```

Comment:
Couldn't find a reason we need this light View wrapper for Reports or Budget anymore. Seems to work as normal without it.